src-tauri/src/main.rs

```diff
@@ -21,13 +21,18 @@ use tracing_subscriber::prelude::*;
 use tracing_subscriber::Layer;
 use tracing_subscriber::{fmt, fmt::format::FmtSpan, EnvFilter};
 
+const KEEP_LAST_N_DAYS: usize = 5;
```

Comment:
Do we need 5 days though? I think keeping just one day would be more than enough. You can increase level of logs though and keep smaller time span

Reply:
I'd rather say 14 days would be preferred, depending on how big the log is

Reply:
i recall the daily logs wont reach GBs. It might be around 10MB maybe, but i'm not 100% sure.

Reply:
I've done some testing, approx 1MB per hour, so 24MB per day. 
What is our final decision on this parameter in this case?